sql/hive-thriftserver/v1.2.1/src/main/java/org/apache/hive/service/auth/HiveAuthFactory.java

```diff
@@ -198,9 +198,6 @@ public TTransportFactory getAuthTransFactory() throws LoginException {
 
   /**
    * Returns the thrift processor factory for HiveServer2 running in binary mode
-   * @param service
```

Comment:
Ah, I forgot to note that we probably don't want to modify this Hive code as it's copied from Hive, and it's easier not to change it. Could you just revert all the changes to these dirs?

Reply:
ok, thanks.